modules/tinymce/src/core/test/ts/browser/fmt/ListItemFormatTest.ts

```diff
@@ -374,4 +374,116 @@ describe('browser.tinymce.core.fmt.ListItemFormatTest', () => {
       );
     });
   });
+
+  context('Apply list formatting', () => {
+    const hook = TinyHooks.bddSetupLight<Editor>({
+      apply_list_formatting: false,
```

Comment:
This is no longer needed? Does the test make sense anymore?

Reply:
this test is testing for list format with indentation, should still be applicable, but in the wrong place. I'll move this test to the `ListIndentationFormatTest.ts`